.github/workflows/verify.yaml

```diff
@@ -0,0 +1,75 @@
+name: verify
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+
+      - name: Install ShellCheck and shfmt
+        run: |
+          sudo apt update
+          sudo apt install -y shellcheck
+          go install mvdan.cc/sh/v3/cmd/shfmt@latest
```

Comment:
Is there a specific reason why we're not installing this through apt along with shellcheck?

Reply:
I tried it but apt can't find the package.

Reply:
That's strange, it definitely exists on the package repository for 22.04: https://packages.ubuntu.com/jammy/shfmt

Reply:
Let me check again .. maybe I was testing it locally and it was failing there.

Reply:
yes you are right its works fine